dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java

```diff
@@ -1,56 +1,71 @@
 package com.dep.Services;
 
 import com.dep.config.BRSConfiguration;
+import com.dep.models.TweetRespons;
 import twitter4j.Status;
 import twitter4j.Twitter;
 import twitter4j.TwitterException;
 import twitter4j.TwitterFactory;
 import twitter4j.conf.ConfigurationBuilder;
-
-import javax.ws.rs.InternalServerErrorException;
+import java.text.Format;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
 
 
 public class TwitterImplement {
-   BRSConfiguration brsConfiguration;
-   ConfigurationBuilder configurationBuilder;
-   TwitterFactory twitterFactory;
-   Twitter twitter;
+    BRSConfiguration brsConfiguration;
+    ConfigurationBuilder configurationBuilder;
+    TwitterFactory twitterFactory;
+    Twitter twitter;
+    TweetRespons tweetRespons;
+
+    public TwitterImplement() {
+        brsConfiguration = new BRSConfiguration();
+        configurationBuilder = brsConfiguration.configurationBuilder();
+        twitterFactory = new TwitterFactory(configurationBuilder.build());
+        twitter = twitterFactory.getInstance();
 
-   public TwitterImplement(){
-       brsConfiguration = new BRSConfiguration();
-       configurationBuilder = brsConfiguration.configurationBuilder();
-       twitterFactory = new TwitterFactory(configurationBuilder.build());
-       twitter = twitterFactory.getInstance();
 
-   }
+    }
 
-   public TwitterImplement(TwitterFactory twitterFactory)
-   {
-       this.twitterFactory=twitterFactory;
-       this.twitter = twitterFactory.getInstance();
-   }
+    public TwitterImplement(TwitterFactory twitterFactory) {
+        this.twitterFactory = twitterFactory;
+        this.twitter = twitterFactory.getInstance();
+    }
 
     public Status sendTweets(String tweet) throws TwitterException {
         Status status;
         status = twitter.updateStatus(tweet);
-
         return status;
     }
 
-    public ArrayList<String> fetchLatestTweets()
-    {
-        ArrayList<String> arrayList = new ArrayList<>();
-        try{
-            List<Status> statuses = twitter.getHomeTimeline();
-            for (int i=0;i<statuses.size();i++){
+
+    public ArrayList<TweetRespons> fetchLatestTweets() {
+        String twitterHandle;
+        String name;
+        String message = null;
+        String profileImageUrl = null;
+        Date createdAt = null;
+
```

Comment:
declaration is not needed, if twitter.getHomeTimeline() does not return anything then these declarations are redundant